src/stylesheets/components/_article.scss

```diff
@@ -0,0 +1,52 @@
+article {
+  margin-bottom: $base-spacing-unit-small;
+  ul, ol {
+    list-style: none;
+  }
+  ul, ol {
```

Comment:
could remove lines 3 - 6, and inline style within this declaration?

Reply:
Hi @naseberry, what do you mean?